core/build.gradle.kts

```diff
@@ -17,4 +17,22 @@ dependencies {
     // https://github.com/InsertKoinIO/koin
     api("io.insert-koin:koin-android:$koinVersion")
     api("io.insert-koin:koin-androidx-workmanager:$koinVersion")
+
+    implementation("io.reactivex.rxjava2:rxjava:2.2.19")
+    implementation("io.reactivex.rxjava2:rxandroid:2.1.1")
+    implementation("com.squareup.retrofit2:retrofit:2.9.0")
+
+    implementation("com.squareup.retrofit2:adapter-rxjava2:2.9.0")
+    implementation("com.squareup.retrofit2:converter-gson:2.9.0")
+    implementation("com.squareup.okhttp3:logging-interceptor:4.9.3")
+    implementation("com.squareup.retrofit2:converter-scalars:2.9.0")
+    implementation("com.google.code.gson:gson:2.9.0")
+
+    val scarlet_version = "0.1.12"
+    implementation("com.tinder.scarlet:scarlet:$scarlet_version")
```

Comment:
what is scarlet?

Reply:
I Don't know, I will remove it's after we will finish bridge, now I just copied dependencies